src/components/ProjectDashboard/components/NftRewardsPanel/NftReward/NftReward.tsx

```diff
@@ -65,7 +65,7 @@ export function NftReward({
   return (
     <div
       className={twMerge(
-        'relative flex h-full w-[252px] select-none flex-col rounded-[10px] border border-grey-200 dark:border-slate-500',
+        'relative flex h-full w-[164px] select-none flex-col rounded-[10px] border border-grey-200 dark:border-slate-500 md:w-[252px]',
```

Comment:
Does `h-40` look good enough? instead of 164px

Reply:
LGETM